front/app/modules/commercial/project_folders/citizen/containers/ProjectFolderShowPage/index.tsx

```diff
@@ -153,33 +155,33 @@ const CardsWrapper = styled.div`
   background: ${colors.background};
 `;
 
+const publicationStatuses: PublicationStatus[] = ['published', 'archived'];
+
 const ProjectFolderShowPage = memo<{
   projectFolder: IProjectFolderData;
 }>(({ projectFolder }) => {
   const authUser = useAuthUser();
   const locale = useLocale();
   const tenant = useAppConfiguration();
-  const {
-    childrenOf: adminPublicationChildrenOf,
-    list: adminPublicationsList,
-  } = useAdminPublicationPrefetchProjects({
-    publicationStatusFilter: ['published', 'archived'],
+
+  const { list: adminPublicationsList } = useAdminPublications({
+    publicationStatusFilter: publicationStatuses,
   });
+
+  const childProjects = useAdminPublicationChildren({
+    publicationId: !isNil(adminPublicationsList)
```

Comment:
Because `adminPublicationList` cannot be an error here. Do you mean that it is weird that it cannot be an error, and that it should have `Error` as one of the possible types here, so that the problem is inside of `useAdminPublications`?

Reply:
The types for list in useAdminPublications and adminPublications in useAdminPublicationChildren should ideally be the same, given they use the same endpoint. We haven't been consistent with this in hooks though I think.

Btw, I wanted to suggest to change childProjects to something different because I thought children could be folders or projects, but it can only be project. See the name is confusing? 😉 Perhaps we should change useAdminPublicationChildren to useFolderProjects or useProjectsInFolder?

Reply:
I mean, the name is confusing because of how projects and folders are modeled throughout the app: both are just called `adminPublications`. I think the reason for talking about "`adminPublications` with children" and "children of `adminPublications`", instead of just folders and projects, is that we might want to create folders with multiple levels in the future (e.g. a folder containing other folders). But in any case, the hook could be called `useFolderChildren`- that way it would also be future-proof.

Reply:
Solve the type issue, but not the hook name yet

Reply:
We have the concept of folders indeed. We have AdminPublicationType. It's currently either a project or folder.

I've looked a bit deeper into it, and I think there's more to it, which will help us solve the name issue (which I think is important to avoid confusion and bugs and the reason why I'm pushing to get it right):

- useFolderChildren would already be clearer and more correct to me. But then to be fully consistent, it should be useProjectFolderChildren, because we name folders 'project folders' in the front-end.
- Which brings me to another point: you can (should?) put this hook in the project_folders module with the other project folder hooks. The hook we're discussing here is only used in the project_folders module.
- You can delete both useAdminPublication and GetAdminPublication, as both are not used. If we need a specific admin publication, we use useProject or useProjectFolder.
- With a light refactor in SiteMap, you also delete GetAdminPublications.
- How I see it: we should only use adminPublications when we need a mix of both projects and folders, which we do when we use the useAdminPublications hook. Otherwise, we use the more specific and leaner projects/projectFolders services.
- If we decide we currently only use it to show folder children (which we do, as projects won't have children), we can decide to name it useProjectFolderChildren (and put it in the project_folders module).
- You can (should?) also use the projects services instead of adminPublications in this hook. They have the same queryParameters/API and using projects here is currently more correct (as we don't have other types of children). If we later decide to add folders as folder children, we can change it to use the adminPublications service.

Reply:
Ignore my comment above, it's full with mistakes. I didn't understand admin publications well enough. Changes coming in my PR.